geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -390,6 +391,10 @@ private void initialize() {
    */
   private SSLContext createAndConfigureSSLContext() throws GeneralSecurityException, IOException {
```

Comment:
I believe this entire getSSLContext logic could be moved into SSLConfig. The config knows all the information to create a SSLContext.